src/NServiceBus.NHibernate.Tests/API/APIApprovals.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace NServiceBus.NHibernate.Tests.API
+{
+    using System.IO;
+    using System.Runtime.CompilerServices;
+    using ApiApprover;
+    using ApprovalTests;
+    using ApprovalTests.Reporters;
+    using Mono.Cecil;
+    using NUnit.Framework;
+
+    [TestFixture]
+    [Explicit("Marking it as explicit before we have all upgrade issue covered in a guide.")]
```

Comment:
Sorry, this was a reminder to me that the tests were still breaking. Of course I forgot to remove it before pushing. I will fix it.